utils/load_unitconv.m

```diff
@@ -14,36 +13,66 @@ function load_unitconv(ringmode, renamedIndexes)
 f_pchip = fopen(pchip_file, 'w');
 uc_id = 0;
 
-fprintf(f_units, 'el_id,field,uc_type,uc_id\n');
+fprintf(f_units, 'el_id,field,uc_type,uc_id,phys_units,eng_units\n');
 fprintf(f_poly, 'uc_id,coeff,val\n');
 fprintf(f_pchip, 'uc_id,eng,phy\n');
 
 quad_families = findmemberof('QUAD');
 sext_families = findmemberof('SEXT');
 
+% Unit conversions for lattice fields
+fprintf(f_units, '%d,%s,poly,%d,%s,%s\n', 0, 'energy', 0, 'Mev', 'Mev');
```

Comment:
Should be `MeV` and `GeV` (`GeV` being whichever is the smaller number)

Reply:
Good catch I will correct the physics units to Gev.